scl/paloalto/panos.conf

```diff
@@ -0,0 +1,97 @@
+#############################################################################
+# Copyright (c) 2020 Balabit
+# created by mileek@gmail.com
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of the GNU General Public License version 2 as published
+# by the Free Software Foundation, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
+#
+# As an additional exemption you are allowed to compile & link against the
+# OpenSSL libraries as published by the OpenSSL project. See the file
+# COPYING for details.
+#
+#############################################################################
+# Requires syslog(transport("udp")) as source
```

Comment:
Can you explain how does this depend on udp?

Reply:
Parser has been tested only when PaloAlto sends logs as RFC3164 (legacy or BSD-syslog) . If you set it to RFC5424 (IETF) then I am not sure how it will behave. I though there should be some  warning in this regards.

Reply:
So this comment is wrong in the opposite way? `syslog` source referns to `RFC5424`.
It would be the `network` source that refers to `RFC3164`.

If the only requirement is the message format, just comment the RFC here, instead of specifying the source. It will help us later include your parser into the app-parser framework.

Reply:
Oh I did not see you modified the patchset in the meantime, and removed the comment. Marking as resolved.